src/Select/select.module.scss

```diff
@@ -1,82 +1,176 @@
   position: relative;
 }
 
-.form-group select {
-  min-width: 150px;
+.select-group {
+  display: flex;
+  flex-direction: column;
+  font-family: $r-regular;
+  position: relative;
+  width: 100%;
+}
+
+.footer {
+  text-align: center;
+  font-size: 14px;
 }
 
-.form-control {
+.select {
+  text-align: start;
+  border-radius: 25px;
+  border: solid 1px #939393;
+  padding: 16px 34px;
+  font-family: $r-regular;
+  font-size: 16px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.25;
+  letter-spacing: normal;
+  color: #1e1e1e;
+  width: 100%;
   -moz-appearance: none !important;
   -webkit-appearance: none !important;
   background: $select-img;
   background-clip: padding-box;
   background-color: $transparent;
-  border: 2px solid $gray-3;
-  border-radius: 12px;
-  box-shadow: inset 0 2px 0 0 rgba(84, 86, 88, 0.06);
-  box-sizing: border-box;
-  color: $gray-11;
-  cursor: pointer;
-  display: block;
-  font-size: 14px;
-  font-weight: 500;
-  line-height: 1.5;
-  padding: 0.375rem 1.35rem 0.375rem 0.75rem;
-  text-indent: 3px;
-  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
-  width: 100%;
+
+  @media screen and (max-width: $tablet) {
+    background-position-x: 95%;
+  }
+
+  @media screen and (max-width: $mobile) {
+    background-position-x: 93%;
+  }
 
   &::placeholder {
     color: $gray-4;
     // Override Firefox's unusual default opacity; see https://github.com/twbs/bootstrap/pull/11526.
     opacity: 1;
   }
 
+  &:focus {
+    border-width: 1px;
+    border-color: lighten($black, 30%);
+    outline: none;
+    transition: all 0.2s ease-out;
+  }
+
+  &:focus + .label,
+  &.has-value + .label {
+    background: $white;
+    top: -8px;
+    left: 33px;
+    font-size: 12px;
+    line-height: 16px;
+    padding-left: 2px;
+    padding-right: 4px;
+  }
+
   &:disabled,
   &[readonly] {
-    background-color: $white;
+    border: solid 1px $gray-3;
+    color: lighten(#1e1e1e, 70%);
     cursor: not-allowed;
     // iOS fix for unreadable disabled content; see https://github.com/twbs/bootstrap/issues/11655.
     opacity: 0.5;
   }
 
-  // sizes
+  &.error {
+    outline: 0;
+    border-color: $red;
+
+    &:disabled {
+      border-color: lighten($red, 25%);
+      color: lighten($red, 30%);
+    }
+  }
+
   &.sm {
-    background-size: 30px;
-    font-size: 14px;
-    height: 30px;
-    padding: 0.25rem 1.25rem 0.25rem 0.5rem;
+    padding: 8px 20px;
   }
 
   &.md {
-    background-size: 34px;
-    height: 40px;
+    padding: 12px 25px;
   }
 
-  &.lg {
-    background-size: 44px;
-    height: 50px;
-    padding: 0px 20px 0px 15px;
+  &:-webkit-autofill,
+  &:-webkit-autofill:hover,
+  &:-webkit-autofill:focus,
+  &:-webkit-autofill:active {
+    -webkit-transition: 'color 9999s ease-out, background-color 9999s ease-out';
+    transition: 'color 9999s ease-out, background-color 9999s ease-out';
+    -webkit-transition-delay: 9999s;
+    transition-delay: 9999s;
```

Comment:
that's a loooooong delay

Reply:
Yeah I noticed that too, but that came straight from the input component.